cloudsmith_cli/cli/tests/commands/policy/test_licence.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+import json
+
+import pytest
+import six
+
+from ....commands.policy.license import create, delete, ls, update
+from ...utils import random_bool, random_str
+
+
+def create_license_policy_config_file(
+    directory,
+    name,
+    description,
+    allow_unknown_licenses,
+    spdx_identifiers,
+    on_violation_quarantine,
+):
+    """Create a license policy config file in `directory` and return its path."""
+
+    data = {
+        "name": name,
+        "description": description,
+        "spdx_identifiers": list(spdx_identifiers),
+        "allow_unknown_licenses": allow_unknown_licenses,
+        "on_violation_quarantine": on_violation_quarantine,
+    }
+
+    file_path = directory / "License-POLICY-CONFIG.json"
```

Comment:
NAB: Could you make the casing consistent on the filename? Doesn't make a bit of difference, but... I'll know it's there.